src/extension/src/azure/utils/index.ts

```diff
@@ -0,0 +1 @@
+export * from "../utils/nameValidator";
```

Comment:
what is this for?

Reply:
Copied the nameValidator from the azure-functions folder and they had a file for exporting, so I just copied that file over too. Will have another PR for addressing all of these issues.